tests/cli/test_train_model.py

```diff
@@ -144,3 +146,62 @@ def test_no_architecture_name(monkeypatch, tmp_path):
         )
     except subprocess.CalledProcessError as captured:
         assert "Architecture name is not defined!" in str(captured.output)
+
+
+@pytest.mark.parametrize("seed", [1234, -1, -123])
```

Comment:
Should we use as default `null`/`None` instead of `-1`? Maybe this is more yaml/pythonic style. Opinions @frostedoyster ?

Reply:
@frostedoyster agreed with you. Now it is done